src/test/java/mocks/decorator/failingpattern/FailingCream.java

```diff
@@ -0,0 +1,26 @@
+package mocks.decorator.failingpattern;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * Should fail predicate because the class does not set it's component at any point
```

Comment:
Is this comment correct? It does set the component in the first constructor

Reply:
Comment changed, nice catch! =D